editor/test/actions.js

```diff
@@ -1,11 +1,6 @@
 /**
  * External dependencies
  */
-import { expect } from 'chai';
-
-/**
- * Internal dependencies
- */
```

Comment:
This section should now be titled `Internal dependencies`, not `External dependencies`.